fiwareskuld/phase/phase0_generateuserlist_community_resources.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+# -- encoding: utf-8 --
+#
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-Core project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+from datetime import datetime
+import os.path
+
+from fiwareskuld.expired_users import ExpiredUsers
+from fiwareskuld.utils import osclients
+from fiwareskuld.conf import settings
+from fiwareskuld.utils import log
+from fiwareskuld.utils import rotated_files
+from fiwareskuld.users_management import UserManager
+from fiwareskuld.conf import settings
+
+__author__ = 'henar'
+
+logger = log.init_logs('phase0')
+
+
+class CommunityUsers:
+    def __init__(self):
+        """Constructor. Create a keystone client"""
+        clients = osclients.OpenStackClients()
+        clients.override_endpoint(
+            'identity', clients.region, 'admin', settings.KEYSTONE_ENDPOINT)
+        self.keystoneclient = clients.get_keystoneclientv3()
+        self.protected = set()
+
+    def generate_community_users_regions(self):
+        users = ExpiredUsers('', '', '')
+
+        community_users = users.get_community_users()
+
+        user_manager = UserManager()
+
+        with open('community_users_regions.txt', 'w') as users_to_delete:
+            for user in community_users:
+                users_to_delete.write(user.id + " " + user.name + " " + user.community_started_at + " ")
+                regions = user_manager.get_regions(user)
+                if regions:
+                    users_to_delete.write(user_manager.get_regions(user))
```

Comment:
Why are you calling again the user_manager.get_regions(user)? you can use regions as well, cannot?